presto-benchmark-runner/src/test/java/com/facebook/presto/benchmark/framework/TestBenchmarkQueryExecutor.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.benchmark.framework;
+
+import com.facebook.presto.benchmark.event.BenchmarkQueryEvent;
+import com.facebook.presto.benchmark.event.BenchmarkQueryEvent.EventStatus;
+import com.facebook.presto.benchmark.prestoaction.JdbcPrestoAction;
+import com.facebook.presto.benchmark.prestoaction.PrestoAction;
+import com.facebook.presto.benchmark.prestoaction.PrestoClusterConfig;
+import com.facebook.presto.benchmark.prestoaction.PrestoExceptionClassifier;
+import com.facebook.presto.benchmark.retry.RetryConfig;
+import com.facebook.presto.jdbc.QueryStats;
+import com.facebook.presto.spi.ErrorCodeSupplier;
+import com.facebook.presto.sql.parser.SqlParser;
+import com.facebook.presto.sql.parser.SqlParserOptions;
+import com.facebook.presto.sql.tree.Statement;
+import com.facebook.presto.tests.StandaloneQueryRunner;
+import com.google.common.collect.ImmutableSet;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.HashMap;
+import java.util.Optional;
+
+import static com.facebook.presto.benchmark.BenchmarkTestUtil.CATALOG;
+import static com.facebook.presto.benchmark.BenchmarkTestUtil.SCHEMA;
+import static com.facebook.presto.benchmark.BenchmarkTestUtil.setupPresto;
+import static com.facebook.presto.benchmark.event.BenchmarkQueryEvent.EventStatus.FAILED;
+import static com.facebook.presto.benchmark.event.BenchmarkQueryEvent.EventStatus.SUCCEEDED;
+import static com.facebook.presto.spi.StandardErrorCode.GENERIC_INTERNAL_ERROR;
+import static com.facebook.presto.sql.parser.IdentifierSymbol.AT_SIGN;
+import static com.facebook.presto.sql.parser.IdentifierSymbol.COLON;
+import static java.util.Objects.requireNonNull;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotNull;
+
+public class TestBenchmarkQueryExecutor
+{
+    private static StandaloneQueryRunner queryRunner;
+    private static final String NAME = "test-query";
+    private static final String TEST_ID = "test-id";
+    private static final String QUERY_SET = "test_set";
+    private static SqlParser sqlParser;
```

Comment:
This should be private final, not static. Initialize the variable here, directly.